src/index.css

```diff
@@ -0,0 +1,126 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px;
+  background-color: black;
+}
+
+.back-arrow {
+  list-style: none;
+  color: red;
+}
+
+a {
+  text-decoration: none;
+  color: red;
+}
+
+.logo-img {
+  width: 45px;
+  height: 40px;
+}
+
+.settings-img {
+  width: 25px;
+  height: 25px;
+}
+
+.settings {
+  color: red;
+}
+
+input {
+  padding: 10px;
+  margin: 10px;
+  border-color: red;
+  border-radius: 5px;
+  color: red;
+}
+
+.games {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  padding: 5px;
+}
+
+.single-game {
+  background-color: black;
+  border-radius: 5px;
+}
+
+.game-title {
+  margin-top: 0;
+  cursor: pointer;
+}
+
+.img-api {
+  padding: 10px;
+  width: 380px;
+  height: 290px;
+  align-self: center;
+}
+
+.game-detail-container {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  align-items: center;
+  padding: 10px;
+}
+
+.game-details {
+  display: flex;
+  flex-direction: column;
+  padding: 5px;
+}
+
+.details-title {
+  color: red;
+}
+
+.details-content {
+  color: black;
+  font-size: 0.9rem;
+}
+
+@media screen and (min-width: 780px) {
```

Comment:
- [ ] As per the given design, in mobile view   there are two columns with alternating colors.  At the moment your app displays the games in just one column.  Please kindly make it two columns.

Reply:
### done

![fixedfeedback](https://user-images.githubusercontent.com/77294268/221010885-8178d880-4b9a-4704-a720-ae14a1e84180.png)